configure.ac

```diff
@@ -194,38 +194,32 @@ AS_IF([test x$enable_staged = xyes], [
   AC_DEFINE(BUILDOPT_STAGE_DEPLOYMENTS, 1, [Define if we are staging deployments])
 ])
 
-AC_ARG_ENABLE(rust,
-              AS_HELP_STRING([--enable-rust],
-                             [Compile Rust features (e.g. compose tree --yaml)]),,
-              [enable_rust=no; rust_debug_release=no])
+AC_PATH_PROG([cargo], [cargo])
+AS_IF([test -z "$cargo"], [AC_MSG_ERROR([cargo is required for --enable-rust])])
+AC_PATH_PROG([rustc], [rustc])
+AS_IF([test -z "$rustc"], [AC_MSG_ERROR([rustc is required for --enable-rust])])
+AC_DEFINE(HAVE_RUST, 1, [Define if we are building with Rust])
```

Comment:
Hmm, we no longer need `HAVE_RUST` though, right? One of the nice benefits of doing this is being able to drop the `#ifdef`s.